tools/releasing/create_source_release.sh

```diff
@@ -75,5 +75,7 @@ tar czf ${PRODUCT_NAME}-src.tgz \
     ${PRODUCT_NAME}
 
 gpg --armor --detach-sig ${PRODUCT_NAME}-src.tgz
+gpg --armor --detach-sig ${PRODUCT_NAME}.tgz
```

Comment:
The file name is `create_source_release.sh`, but these two lines are signing binary package, what do you think moving these two lines to the release doc or renaming this script file?

Reply:
I prefer rename the script file.

Reply:
@kezhenxu94 I updated the docs a little bit more, to move binary into releasing folder.